packages/amplify-codegen/src/commands/models.js

```diff
@@ -72,6 +73,22 @@ async function generateModels(context) {
   const generateIndexRules = readFeatureFlag('codegen.generateIndexRules');
   const emitAuthProvider = readFeatureFlag('codegen.emitAuthProvider');
 
+  let enableDartNullSafety = readFeatureFlag('codegen.enableDartNullSafety');
```

Comment:
Will this feature flag automatically get added to their cli.json prior to this code getting hit?

Reply:
Yes. If they are using the right CLI version (equal to above the version when this FF is added)

Reply:
They will be added ahead as long as they are registered in CLI (aka Phani's PR gets merged and released)